src/unix/solarish/mod.rs

```diff
@@ -3202,6 +3202,21 @@ extern "C" {
     pub fn arc4random_uniform(upper_bound: u32) -> u32;
 
     pub fn secure_getenv(name: *const c_char) -> *mut c_char;
+
+    #[link_name = "__xpg4_strftime"]
```

Comment:
Doing this unconditionally for both Solaris and illumos is going to break the latter, I believe.

Reply:
in fact I m just trying to see if it fixes solaris case :) thus this PR being a draft for now